README.md

```diff
@@ -1035,6 +1035,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Sarvesh Kumar Dwivedi](https://github.com/sarvesh4396)
   - [Sarvesh Patil](https://github.com/sarveshspatil111)
   - [Sarwar Ahmed](https://github.com/SarwarAhmed)
+  - [Sattyam Samania] (https://github.com/sam2810)
```

Comment:
hey @sam2810, remove the space after the square bracket. 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix